src/algorithm.erl

```diff
@@ -5,21 +5,21 @@
 -include("parallant.hrl").
 
 
-% Callbacks
+%% Callbacks
 
 -callback test() ->
     ok.
 
 -callback run(Steps, environment(), config()) -> environment()
-        when Steps :: pos_integer().
+                                                     when Steps :: pos_integer().
```

Comment:
Line 14 is too long: "                                                     when Steps :: pos_integer().".